01-messaging-crypto-and-init.md

```diff
@@ -20,24 +20,21 @@ the unicode code point for LIGHTNING.<sup>[2](#reference-2)</sup>
 
 All messages are of form:
 
-1. `4-byte` big-endian data length.
-2. `4-byte` big-endian type.
-3. Data bytes as specified by the length.
+1. `2-byte` big-endian type.
+2. 4 bytes of padding.
```

Comment:
It does if we decrypt-in-place.  But I agree that's unlikely; see above.

My unmarshalling implementation doesn't actually care about alignment, though it could be optimized a little if it did.